src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java

```diff
@@ -81,4 +89,39 @@ public Date deserialize(JsonElement jsonElement, Type type,
             }
         }
     }
+
+    /**
+     * Deserializer for the Stats object received from the API.
+     */
+    private class StatsDeserializer implements JsonDeserializer<Stats> {
+
+        private static final String CP_APPLICATIONS = "CherryPy Applications";
+        private static final String CP_SERVER_PREFIX = "CherryPy HTTPServer ";
+
+        @Override
+        public Stats deserialize(JsonElement jsonElement, Type type,
+                JsonDeserializationContext jsonDeserializationContext)
+                throws JsonParseException {
+            try {
+                JsonObject stats = jsonElement.getAsJsonObject();
+                Applications app = gson.fromJson(stats.get(CP_APPLICATIONS), Applications.class);
+                HttpServer server = null;
+                for(Map.Entry<String, JsonElement> entry : stats.entrySet()){
+                    String key = entry.getKey();
+                    if(key.startsWith(CP_SERVER_PREFIX)){
```

Comment:
Let's put blanks after `for` and `if` as well as before the `{` (throughout the whole patch).